presto-main/src/test/java/com/facebook/presto/cost/TestCoefficientBasedCostCalculator.java

```diff
@@ -1,88 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.cost;
-
-import com.facebook.presto.spi.statistics.Estimate;
-import com.facebook.presto.sql.planner.LogicalPlanner;
-import com.facebook.presto.sql.planner.Plan;
-import com.facebook.presto.sql.planner.assertions.PlanAssert;
-import com.facebook.presto.sql.planner.assertions.PlanMatchPattern;
-import com.facebook.presto.sql.planner.plan.FilterNode;
-import com.facebook.presto.sql.planner.plan.TableScanNode;
-import com.facebook.presto.testing.LocalQueryRunner;
-import com.facebook.presto.tpch.TpchConnectorFactory;
-import com.google.common.collect.ImmutableMap;
-import org.testng.annotations.Test;
-
-import static com.facebook.presto.spi.statistics.Estimate.unknownValue;
-import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.anyTree;
-import static com.facebook.presto.sql.planner.assertions.PlanMatchPattern.node;
-import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
-
-public class TestCoefficientBasedCostCalculator
```

Comment:
There are more high level tests in @kokosing's PR:
https://github.com/Teradata/presto/pull/530/commits/b4473b1e399380199fddd1da84903ed47a4c1bbc